aws_lambda_powertools/event_handler/api_gateway.py

```diff
@@ -1559,6 +1559,7 @@ def enable_swagger(
         license_info: Optional["License"] = None,
         swagger_base_url: Optional[str] = None,
         middlewares: Optional[List[Callable[..., Response]]] = None,
+        enable_download_spec: Optional[bool] = False,
```

Comment:
```suggestion
        enable_download_spec: bool = False,
```

False is the default value, so, we don't need `Optional` in this field.